From 1b457a908b27d20558653f2deae725ce4776a7b4 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Fri, 4 Dec 2015 18:05:28 +0800
Subject: [PATCH 0767/5242] MLK-11953: arm: dts: Fix HDMI compliance test 7-15
 failed

commit  d1d12a1d2f68fca351b05291316e77fe8bca59b4 from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX6QP SabreSD board HDMI compliance test 7-15 Vcec2 test failed,
the value is 0.403V, which should be 0.274V>Vcec2>0.196V.

Disable the pad internal PU resistor, and change the external PU
resistor(R647) from 47K to 100K.
With all the changes, the calculated voltage on CEC is 0.248V, and the
test value is 0.250V.

In the patch disable CEC pin internal PULL/Keep function.

Signed-off-by: Sandor Yu <R01008@freescale.com>
(cherry picked from commit 0a9388ff8ef77778f5e81d8dbc980cacf87fc12b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index c6cc9f3..66fd76c 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -771,7 +771,7 @@
 
 		pinctrl_hdmi_cec: hdmicecgrp {
 			fsl,pins = <
-				MX6QDL_PAD_KEY_ROW2__HDMI_TX_CEC_LINE 0x1f8b0
+				MX6QDL_PAD_KEY_ROW2__HDMI_TX_CEC_LINE 0x108b0
 			>;
 		};
 
-- 
1.7.9.5

